<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPage</class>
 <widget class="QWizardPage" name="WizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Data source selection</string>
  </property>
  <property name="subTitle">
   <string>Please select the source of your data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="sourceFromWFS">
     <property name="text">
      <string>&amp;WFS</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="sourceFromFile">
       <property name="text">
        <string>File/&amp;URL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="gmlPathLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="gmlPathButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sourceFromFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>gmlPathLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sourceFromFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>gmlPathButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
